[2025-02-23T12:42:01.653+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-23T12:42:01.682+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: newyork_weather_pipeline.transform_weather_data manual__2025-02-23T12:41:58.269055+00:00 [queued]>
[2025-02-23T12:42:01.691+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: newyork_weather_pipeline.transform_weather_data manual__2025-02-23T12:41:58.269055+00:00 [queued]>
[2025-02-23T12:42:01.692+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-23T12:42:01.708+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): transform_weather_data> on 2025-02-23 12:41:58.269055+00:00
[2025-02-23T12:42:01.714+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=18440) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-23T12:42:01.715+0000] {standard_task_runner.py:72} INFO - Started process 18441 to run task
[2025-02-23T12:42:01.716+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'newyork_weather_pipeline', 'transform_weather_data', 'manual__2025-02-23T12:41:58.269055+00:00', '--job-id', '109', '--raw', '--subdir', 'DAGS_FOLDER/newyork_weather_pipeline/newyork_weather_etl.py', '--cfg-path', '/tmp/tmpca62u8jp']
[2025-02-23T12:42:01.717+0000] {standard_task_runner.py:105} INFO - Job 109: Subtask transform_weather_data
[2025-02-23T12:42:01.782+0000] {task_command.py:467} INFO - Running <TaskInstance: newyork_weather_pipeline.transform_weather_data manual__2025-02-23T12:41:58.269055+00:00 [running]> on host ee2081ae3a04
[2025-02-23T12:42:01.922+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='newyork_weather_pipeline' AIRFLOW_CTX_TASK_ID='transform_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-23T12:41:58.269055+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-23T12:41:58.269055+00:00'
[2025-02-23T12:42:01.923+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-23T12:42:01.926+0000] {python.py:240} INFO - Done. Returned value was: {'latitude': '40.7128', 'longitude': '74.0060', 'temperature': -8.9, 'windspeed': 9.4, 'winddirection': 72, 'weathercode': 2}
[2025-02-23T12:42:01.966+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-23T12:42:01.967+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=newyork_weather_pipeline, task_id=transform_weather_data, run_id=manual__2025-02-23T12:41:58.269055+00:00, execution_date=20250223T124158, start_date=20250223T124201, end_date=20250223T124201
[2025-02-23T12:42:02.050+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-23T12:42:02.091+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-23T12:42:02.095+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
